app/decorators/listing_decorator.rb

```diff
@@ -1,70 +1,36 @@
-class ListingDecorator
-  def initialize(listing)
-    @listing = listing
+class ListingDecorator < OpenStruct
+  CONDITION_COLORS = {
+                        "Like New" => "00FF00",
+                        "Mint" => "72DF3B",
+                        "Excellent" => "008C00",
+                        "Very Good" => "005000",
+                        "Good" => "005500",
+                        "Fair" => "#FFFF00",
+                        "Poor" => "#FFFF00"
+                     }
```

Comment:
Put the actual color codes in CSS and then just add a class that references them via the decorator.